Условные операторы 1/Условные операторы 1/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Условные_операторы_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.Write("Введите координаты x=");
+                int x = Convert.ToInt16(Console.ReadLine());
+                Console.Write("Введите координаты y=");
+                int y = Convert.ToInt16(Console.ReadLine());
+
+                if ((x > 0) && (y > 0))
```

Comment:
В таких логических выражениях лишние скобки ни к чему. Они только затрудняют чтение кода (хотя выражение составлено верно).
```suggestion
                if (x > 0 && y > 0)
```